sklearn/linear_model/stochastic_gradient.py

```diff
@@ -1435,7 +1435,7 @@ class SGDRegressor(BaseSGDRegressor):
 
     eta0 : double
         The initial learning rate for the 'constant', 'invscaling' or
-        'adaptive' schedules. The default value is 0.0 as eta0 is not used by
+        'adaptive' schedules. The default value is 0.01 as eta0 is not used by
```

Comment:
The sentence now sounds kinda odd. Maybe remove the part after 0.01?

Reply:
Sure. I actually wasn't sure how the default value of eta0 followed from the fact that is not used by the default schedule ‘optimal’ so I left the rest of the line as it was. I'll remove it now.

Reply:
@aaron-alphonsus 	you didn't remove this one

Reply:
Yeah, I messed up and removed the line in SGDClassifier instead of SGDRegressor. Is it correct now?